doc(release): prepare release notes for Centreon MAP 22.10.1 (#1990)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md

Retrouvez plus de détails sur la version 22.10 dans notre [post de blog](https://www.centreon.com/blog/centreon-22-10-les-nouveautes).

## Centreon MAP 

### 22.10.1

- First release

> 

Reviewer: Web editor
New Server
Migration process

## Centreon MAP Legacy

### 22.10.1

Release date:

#### Security fixes

- Actuator endpoints are now disabled by default, except for health and metrics
- Fix the security issue CVE-2022-42889 (Text4shell)

#### Bug fixes

> - Fixed an issue where resources tooltips in MAP legacy widget could not be displayed properly

Reviewer 2: ```suggestion
- Fixed an issue where resources' tooltips in the MAP legacy widget could not be displayed properly
```

## Centreon MAP

### 22.10.1

- First release: `<release_date>`

> 

Reviewer 3: ```suggestion

> The new MAP extension is now available in a full web version with a new server, under the name of MAP. The former version, including desktop client and associated server, becomes MAP Legacy.

```

> Web editor: Create and edit your views directly from your web browser

Reviewer 3: ```suggestion
- Web editor: Create and edit your views directly from your web browser.
```

> New server: Brand new server and data model providing better performance

Reviewer 3: ```suggestion
- New server: Brand new server and data model providing better performance.
```

> Migration process: Integrated migration process of your legacy views

Reviewer 3: ```suggestion
- Migration process: Integrated migration process of your legacy views.
```

## Centreon MAP

### 22.10.1

- First release: `<release_date>`

> > The new MAP extension is now available in a full web version with a new server, under the name of MAP. The former version, including desktop client and associated server, becomes MAP Legacy.

Reviewer 3: ```suggestion
The new MAP extension is now available in a full web version with a new server, under the name of MAP. The former version, including desktop client and associated server, becomes MAP Legacy.
```

- Web editor: Create and edit your views directly from your web browser.
- New server: Brand new server and data model providing better performance.
- Migration process: Integrated migration process of your legacy views.

## Centreon MAP Legacy

### 22.10.1

Release date: `November 29, 2022`

#### Security fixes

> - Actuator endpoints are now disabled by default, except for health and metrics

Reviewer 3: ```suggestion
- Actuator endpoints are now disabled by default, except for health and metrics.
```

> - Fix the security issue CVE-2022-42889 (Text4shell)

Reviewer 3: ```suggestion
- Fixed the security issue CVE-2022-42889 (Text4shell).
```

#### Bug fixes

> - Fixed an issue where resources' tooltips in the MAP legacy widget could not be displayed properly

Reviewer 3: ```suggestion
- Fixed an issue where resources' tooltips in the MAP legacy widget could not be displayed properly.
```
